packages/core/src/interfaces/components.ts

```diff
@@ -49,6 +49,12 @@ export interface LegendOptions {
 		radius?: Number;
 		spaceAfter?: Number;
 	};
+	truncation?: {
```

Comment:
`truncation` should move out into it's own interface, because ideally we'd want everything to be able to be truncated using the similar configs. once we have an interface for `truncation` we can add the same options into axes for instance